test-app/app_imx_rt.c

```diff
@@ -124,6 +124,12 @@ void rt1050_init_pins(void)
 
 void main(void)
 {
+    hal_init();
```

Comment:
```make[1]: Entering directory '/home/davidgarske/GitHub/wolfboot/test-app'
	[CC-ARM] app_imx_rt.o
app_imx_rt.c: In function 'main':
app_imx_rt.c:127:5: warning: implicit declaration of function 'hal_init' [-Wimplicit-function-declaration]
  127 |     hal_init();
      |     ^~~~~~~~```

Reply:
it actually wasn't calling `hal_init` that was required, I needed to move the `wolfBoot_update_trigger` to before the SysTick_Config call, it seems to mess with `hal_flash_write`